interop/cats/src/main/scala/caliban/interop/cats/implicits/package.scala

```diff
@@ -37,6 +38,6 @@ package object implicits {
       CatsInterop.interpreterAsync(underlying)
   }
 
-  implicit def catsEffectSchema[F[_]: Effect, R, A](implicit ev: Schema[R, A]): Schema[R, F[A]] =
+  implicit def catsEffectSchema[F[_]: Async, R, A](implicit F: Dispatcher[F], ev: Schema[R, A]): Schema[R, F[A]] =
```

Comment:
`Async` can be removed if I understood properly?

Reply:
Yep. Done